base/missing.jl

```diff
@@ -259,6 +259,14 @@ keys(itr::SkipMissing) =
     v
 end
 
+"""
+    hasmissing(itr)
+
+Checks if itr has missing values.
+"""
+hasmissing(itr) = any(ismissing, itr)
```

Comment:
Maybe consider this?
```suggestion
hasmissing(itr) = eltype(itr) >: Missing && any(ismissing, itr)
```
as `any(ismissing, itr)` does not seem to always optimize-out iterating `itr` even if `eltype(itr) >: Missing` is `false` (sometimes it does).

Reply:
Yeah this is good.